### PR TITLE
Add to Artifactory

### PR DESCRIPTION
**Pull request by omervk**
This pull request:
1. Publishes to Artifactory.
1. Changes the module's name from `eslint-config-bigpanda` to the standard `@bp/eslint-config`.
1. Adds information on how to use this module to `README.md`.

Note that nothing happens in Travis with each commit. Only tagged versions, which get deployed to Artifactory, start a job on Travis.

### CODE REVIEW

**Comment by rinero-bigpanda** [comment]
@omervk This is a **_public_** repository....
image
image

We should make it private before publishing to our internal artifactory or avoid doing it altogether.
I prefer making it private, but I'm not sure what are the consequences :)

Alternatively, we can archive this repo, and create a new private one and keep this for legacy reasons 🤷

**Comment by rinero-bigpanda** [comment]
Also... [pull requests builds on travis triggered by forks can expose secrets](https://docs.travis-ci.com/user/pull-requests#pull-requests-and-security-restrictions)

**Comment by omervk** [comment]
There are no secrets in this repo (they're all injected via the environment), but you're generally right. The only public package currently using this project is [mongoose-patch](https://www.npmjs.com/package/mongoose-patch) ([repo](https://github.com/bigpandaio/mongoose-patch)) and I don't think anyone's using it. I don't think even *we're* using it.

However, to preserve it for the future, I recommend we fork it to a private repository and continue there. WDYT all?

**Comment by omervk** [comment]
> Also... [pull requests builds on travis triggered by forks can expose secrets](https://docs.travis-ci.com/user/pull-requests#pull-requests-and-security-restrictions)

TIL. Holy crap. Luckily, in this case, we don't run Travis builds on each commit.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-  "name": "eslint-config-bigpanda",
-  "version": "5.4.0",
+  "name": "@bp/eslint-config",
+  "version": "0.0.0",
   "private": true,
```
Comment (maria-os) [inline package.json:4]: This should be removed as well

**Comment by omervk** [comment]
I've force pushed an empty commit to the branch. Closing.

**Pull request closed without merging**